jgiven-gradle-plugin/src/main/java/com/tngtech/jgiven/gradle/JGivenPlugin.java

```diff
@@ -72,9 +73,10 @@ private void addDefaultReports( final Project project ){
 
     private void configureDefaultReportTask( final Test test, JGivenReportTask reportTask,
             final ReportingExtension reportingExtension ){
+        reportTask.dependsOn(test);
```

Comment:
change marker :)

Reply:
Ok, adding this fix the tests based on https://docs.gradle.org/7.0/userguide/validation_problems.html#implicit_dependency ?

Reply:
yes